drivers/openstack/openstack.go

```diff
@@ -175,13 +173,6 @@ func GetCreateFlags() []cli.Flag {
 			Usage: "OpenStack SSH port",
 			Value: 22,
 		},
-		// Using a StringFlag rather than a BoolFlag because
-		// the BoolFlag default value is always false
-		cli.StringFlag{
-			Name:  "openstack-docker-install",
```

Comment:
Yes, a global flag would be great. I originally did it in the OpenStack driver because it was not possible to make it global but with your refactoring now it is :+1: